packages/core/src/application.ts

```diff
@@ -61,9 +63,74 @@ export class Application extends Context {
    */
   controller(controllerCtor: ControllerClass, name?: string): Binding {
     name = name || controllerCtor.name;
-    return this.bind(`controllers.${name}`)
+    return this.bind(`${CoreBindings.CONTROLLERS}.${name}`)
       .toClass(controllerCtor)
-      .tag('controller');
+      .tag(CoreBindings.CONTROLLERS_TAG);
+  }
+
+  /**
+   * Register a booter class / array of classes with this application.
+   *
+   * @param booterCls {Function | Function[]} The booter class (constructor function).
```

Comment:
Incorrect ordering; should be typedef, then name
`@param {Function | Function[]} booterCls`

Technically, you can omit the type definition since it's TypeScript and can infer it for you on build (and because it's not an optional param)